packages/ui/components/divider/Divider.tsx

```diff
@@ -1,6 +1,11 @@
 import { classNames } from "@calcom/lib";
 
-export default function VerticalDivider({ className, ...props }: JSX.IntrinsicElements["svg"]) {
+export function Divider({ className, ...props }: JSX.IntrinsicElements["hr"]) {
```

Comment:
Merged the divider and vertical divider into one file. Makes us have less files 🙂

Reply:
Agreed - no reason to have two divider files - especially when the components are this small.